.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,5 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "features": {
+  }
```

Comment:
Probably need to add some features! At least python/conda. Probably rust? I don't use VS Code so probably cannot help beyond that